HashRegistrar.sol

```diff
@@ -148,17 +164,22 @@ contract Registrar {
     function startAuction(bytes32 _hash) noEther {
         entry newAuction = entries[_hash];
         if ((newAuction.status == Mode.Owned && now < newAuction.renewalDate) 
-            || (newAuction.status == Mode.Auction && now < newAuction.auctionExpiration))
+            || (newAuction.status == Mode.Auction && now < newAuction.registrationDate))
             throw;
         
         if (newAuction.status == Mode.Owned) {
             Deed deedContract = Deed(newAuction.deed);
             deedContract.closeDeed(999);
         }
         
-        newAuction.auctionExpiration = now + auctionLength;
+        // for the first six months of the registry, make longer auctions
+        // TODO: change 'hours' into 'weeks' for real contract
+        uint slowStart = 1 + (registryCreated + 20 hours - now) / 4 hours;
```

Comment:
Would a simple threshold scheme not work here? Eg, auctions last 2 weeks for the first 6 months, then revert to 1 week (overnight)?